### ISSUE TITLE
drush entity-updates not working on Windows?

### ISSUE THREAD

**Issue by vibrasphere** [drush-ops/drush#2120]
I am having this problem: https://www.drupal.org/node/2601762#comment-10743216 with pathauto installed I get (in status report):
```
Mismatched entity and/or field definitions
The following changes were detected in the entity type and field definitions.
Content

    Delete the URL alias field.

Taxonomy term

    Delete the URL alias field.
```

And in that thread everyone is reporting that running drush command `drush entity-updates` fix it. But it doesn't for me, I get error "`The filename, directory name, or volume label syntax is incorrect.`"

I tried removing and reinstalling drush, I downgraded from 9-dev version to 8.0.5, didn't fix.

Drupal version 8.0.6
Setup: Win10x64 XAMPP

drush entity-updates --debug:
```
$ drush entity-updates --debug
Phar detected. Proceeding to drush_main().Cache HIT cid: 8.0.5-commandfiles-0-95bcd8283f3ecd8611cba5beee862a9d [0.41 sec, 1.81 MB]                                                                                                         [debug]
Bootstrap to phase 0. [1.32 sec, 5.23 MB]                                                                                                                                                    [bootstrap]
Bootstrap to phase 5. [1.33 sec, 5.24 MB]                                                                                                                                                    [bootstrap]
Drush bootstrap phase : bootstrap_drupal_root() [1.35 sec, 5.66 MB]                                                                                                                          [bootstrap]
Initialized Drupal 8.0.6 root directory at C:\xampp\htdocs\d8 [1.45 sec, 5.66 MB]                                                                                                            [bootstrap]
Find command files for phase 1 (max=5) [1.45 sec, 4.01 MB]                                                                                                                                       [debug]
Cache HIT cid: 8.0.5-commandfiles-1-1455d3419b7a495be52bbf211a77b39a [1.46 sec, 4.01 MB]                                                                                                         [debug]
Drush bootstrap phase : bootstrap_drupal_site() [1.81 sec, 4.62 MB]                                                                                                                          [bootstrap]
Initialized Drupal site default at sites/default [1.81 sec, 4.62 MB]                                                                                                                         [bootstrap]
Find command files for phase 2 (max=5) [1.91 sec, 4.62 MB]                                                                                                                                       [debug]
Drush bootstrap phase : bootstrap_drupal_configuration() [1.91 sec, 4.62 MB]                                                                                                                 [bootstrap]
Find command files for phase 3 (max=5) [1.92 sec, 4.76 MB]                                                                                                                                       [debug]
sql-query: SELECT 1; [1.93 sec, 4.84 MB]                                                                                                                                                        [notice]
Executing: mysql --defaults-extra-file="C:\\Users\\user\\AppData\\Local\\Temp\\druBEBA.tmp" --database=d8 --host=localhost --port=3306 --silent  < "C:\\Users\\user\\AppData\\Local\\Temp\\druBEB9.tmp"
  mysql: Unknown OS character set 'cp775'.
  mysql: Switching to the default character set 'latin1'.
  1
sql-query: SHOW TABLES; [2.04 sec, 4.84 MB]                                                                                                                                                     [notice]
Executing: mysql --defaults-extra-file="C:\\Users\\user\\AppData\\Local\\Temp\\druBF29.tmp" --database=d8 --host=localhost --port=3306 --silent  < "C:\\Users\\user\\AppData\\Local\\Temp\\druBF28.tmp"
  mysql: Unknown OS character set 'cp775'.
  mysql: Switching to the default character set 'latin1'.
  batch
  block_content
  block_content__body
  block_content_field_data
  block_content_field_revision
  block_content_revision
  block_content_revision__body
  cache_bootstrap
  cache_config
  cache_container
  cache_data
  cache_default
  cache_discovery
  cache_dynamic_page_cache
  cache_entity
  cache_hacked
  cache_menu
  cache_render
  cache_toolbar
  cachetags
  config
  field_deleted_data_1232a9fd91
  file_managed
  file_usage
  flood
  history
  key_value
  key_value_expire
  menu_link_content
  menu_link_content_data
  menu_tree
  node
  node__body
  node_access
  node_field_data
  node_field_revision
  node_revision
  node_revision__body
  queue
  router
  semaphore
  sequences
  sessions
  shortcut
  shortcut_field_data
  shortcut_set_users
  taxonomy_index
  taxonomy_term_data
  taxonomy_term_field_data
  taxonomy_term_hierarchy
  url_alias
  user__roles
  user__user_picture
  users
  users_data
  users_field_data
  watchdog
Drush bootstrap phase : bootstrap_drupal_database() [2.26 sec, 4.84 MB]                                                                                                                      [bootstrap]
Successfully connected to the Drupal database. [2.26 sec, 4.84 MB]                                                                                                                           [bootstrap]
Find command files for phase 4 (max=5) [2.26 sec, 4.85 MB]                                                                                                                                       [debug]
Drush bootstrap phase : bootstrap_drupal_full() [2.26 sec, 4.85 MB]                                                                                                                          [bootstrap]
Cache HIT cid: 8.0.5-install_profile-66ecfeb9791a023150773849f1550c5d [2.4 sec, 9.3 MB]                                                                                                          [debug]
Find command files for phase 5 (max=5) [2.41 sec, 9.3 MB]                                                                                                                                        [debug]
Cache HIT cid: 8.0.5-install_profile-66ecfeb9791a023150773849f1550c5d [2.41 sec, 9.31 MB]                                                                                                        [debug]
Cache HIT cid: 8.0.5-commandfiles-5-428c76cd35773804b8e6c77d3ec0fcb6 [2.45 sec, 10.1 MB]                                                                                                         [debug]
Found command: entity-updates (commandfile=core) [2.45 sec, 10.15 MB]                                                                                                                        [bootstrap]
Calling hook drush_core_entity_updates [2.75 sec, 12.59 MB]                                                                                                                                      [debug]
The following updates are pending:

node entity type :
  Delete the URL alias field.
taxonomy_term entity type :
  Delete the URL alias field.
Do you wish to run all pending updates? (y/n): y
Cache MISS cid: 8.0.5-alias-path--be8b1601f448b989b8868b9c4b21f13f [5.08 sec, 22.78 MB]                                                                                                          [debug]
Backend invoke: "C:\\Users\\user\\AppData\\Roaming\\Composer\\vendor\\drush\\drush\\drush"  --backend=2 --user=0 --verbose --debug --root="C:\\xampp\\htdocs\\d8" --uri=http://default           [command]
updatedb-batch-process 21 2>&1 [5.43 sec, 22.81 MB]
"C:\\Users\\user\\AppData\\Roaming\\Composer\\vendor\\drush\\drush\\drush"  --backend=2 --user=0 --verbose --debug --root="C:\\xampp\\htdocs\\d8" --uri=http://default  updatedb-batch-process    [notice]
21 2>&1 [5.43 sec, 22.81 MB]
The filename, directory name, or volume label syntax is incorrect.
                                                                    [error]ectory name, or volume label syntax is incorrect.
, code: 1) [5.63 sec, 22.81 MB]
Cache MISS cid: 8.0.5-alias-path--be8b1601f448b989b8868b9c4b21f13f [5.64 sec, 22.8 MB]                                                                                                           [debug]
Backend invoke: "C:\\Users\\user\\AppData\\Roaming\\Composer\\vendor\\drush\\drush\\drush"  --backend=2 --verbose --debug --root="C:\\xampp\\htdocs\\d8" --uri=http://default  cache-rebuild     [command]
2>&1 [5.98 sec, 22.83 MB]
"C:\\Users\\user\\AppData\\Roaming\\Composer\\vendor\\drush\\drush\\drush"  --backend=2 --verbose --debug --root="C:\\xampp\\htdocs\\d8" --uri=http://default  cache-rebuild 2>&1 [5.98 sec,      [notice]
22.83 MB]
The filename, directory name, or volume label syntax is incorrect.
                                                                    [error]ectory name, or volume label syntax is incorrect.
, code: 1) [6.18 sec, 22.83 MB]
Finished performing updates. [6.18 sec, 22.82 MB]                                                                                                                                                   [ok]
Returned from hook drush_core_entity_updates [6.18 sec, 22.83 MB]                                                                                                                                [debug]
Command dispatch complete [6.19 sec, 22.8 MB]                  
```

![with-verbose](https://cloud.githubusercontent.com/assets/5372918/14349897/caa3c532-fcce-11e5-9c67-78569218130b.png)

drush status:
```
$ drush status
 Drupal version                  :  8.0.6
 Site URI                        :  http://default
 Database driver                 :  mysql
 Database hostname               :  localhost
 Database port                   :  3306
 Database username               :  root
 Database name                   :  d8
 Drupal bootstrap                :  Successful
 Drupal user                     :
 Default theme                   :  bartik
 Administration theme            :  adminimal_theme
 PHP configuration               :  C:\xampp\php\php.ini
 PHP OS                          :  WINNT
 Drush script                    :  C:\Users\user\AppData\Roaming\Composer\vendor\drush\drush\drush
 Drush version                   :  8.0.5
 Drush temp directory            :  C:/Users/user/AppData/Local/Temp
 Drush configuration             :
 Drush alias files               :
 Install profile                 :  standard
 Drupal root                     :  C:\xampp\htdocs\d8
 Drupal Settings File            :  sites/default/settings.php
 Site path                       :  sites/default
 File directory path             :  sites/default/files
 Private file directory path     :  sites/default/files/prv
 Temporary file directory path   :  C:\xampp\tmp
 Sync config path                :  sites/default/files/config_n81RNFJNongpYF-jZtxZRcA7pNLQagsO2UW6EdLRqhm_IsQhIdZ6Hr6T0BVkVnB2omLVb-hvlw/sync

```

**Comment by divined** [comment]
drush updb too

**Comment by weitzman** [comment]
Fixed in #2312. Please post here if I'm wrong (or I'm right).
